### PR TITLE
Update options.py

### PR DESCRIPTION
**Pull request by Mehak-4545**
- [ ] Closes #xxxx
- [ ] Tests added
- [ ] User visible changes (including notable bug fixes) are documented in `whats-new.rst`
- [ ] New functions/methods are listed in `api.rst`

### CODE REVIEW

File: xarray/core/options.py [hunk 1]
```diff
@@ -292,7 +292,11 @@ def __exit__(self, type, value, traceback):
 
 def get_options():
     """
-    Get options for xarray.
+    Retrieves the current global option for xarray
+
+    Returns
+    -------
+    dictionary containing current global options and their corresponding values
```
Comment (headtr1ck) [inline xarray/core/options.py:299]: Usually we use the format:
```
options : FrozenDict
    Mapping from global options to their current values.
```

File: xarray/core/options.py [hunk 2]
```diff
@@ -292,7 +292,11 @@ def __exit__(self, type, value, traceback):
 
 def get_options():
     """
-    Get options for xarray.
+    Retrieves the current global option for xarray
```
Comment (headtr1ck) [inline xarray/core/options.py:295]: Retrieve the current global options for xarray.

File: xarray/core/options.py [hunk 3]
```diff
@@ -292,7 +292,11 @@ def __exit__(self, type, value, traceback):
 
 def get_options():
```
Comment (headtr1ck) [inline xarray/core/options.py:293]: If you are up for it, you could add type hints here. Not sure exactly what but something along:
`def get_options() -> FrozenDict[str, Any]:`

File: xarray/core/options.py (hunk 2 above)
Comment (Mehak-4545) [reply xarray/core/options.py:295]: Thanks for the correction, I'll update it!

File: xarray/core/options.py (hunk 3 above)
Comment (Mehak-4545) [reply xarray/core/options.py:293]: Sure, I'll look into it!

**Comment by Mehak-4545** [comment]
@dcherian I've modified the parameters for get_options() as per changes suggested. Could you review these once, so that I can do the same for other functions as well? Thanks!

File: xarray/core/options.py [hunk 4]
```diff
@@ -290,9 +290,16 @@ def __exit__(self, type, value, traceback):
         self._apply_update(self.old)
 
 
-def get_options():
+def get_options(*, join: str = 'inner', copy: bool = True,
+                indexes: Tuple[str, ...] = None,
+                exclude: FrozenSet[str] = frozenset(),
+                fill_value: Any = None) -> FrozenDict[str, Any]):
```
Comment (headtr1ck) [inline xarray/core/options.py:296]:
```suggestion
                fill_value: Any = None) -> FrozenDict[str, Any]:
```
But actually I have no idea what is the purpose of these arguments? They are not used anywhere!

File: xarray/core/computation.py [hunk 5]
```diff
@@ -1947,7 +1947,29 @@ def polyval(
     Returns
     -------
     DataArray or Dataset
-        Evaluated polynomial.
+    Evaluated polynomial.
+
+
+    Example
```
Comment (headtr1ck) [inline xarray/core/computation.py:1953]:
```suggestion
    Examples
```
Your example is formatted wrong, see other methods as a baseline on how to format.

**Review by headtr1ck — CHANGES_REQUESTED** [review]
Something seems wrong with these changes.

**Comment by Mehak-4545** [comment]
I interpreted the previous change suggested as a suggestion to include all possible keyword arguments for get_options() function, as would be ideal in its documentation, and modified accordingly. Did you mean to suggest something else? I'm sorry if i interpreted it wrongly. Could you explain it again?
Really sorry for the inconvenience!

**Pull request closed without merging**